src/Admin/AlbumAdmin.php

```diff
@@ -23,13 +24,16 @@ class AlbumAdmin extends Admin
     const EDIT_FORM_DETAILS_VIEW = 'app.album.edit_form.details';
 
     private ViewBuilderFactoryInterface $viewBuilderFactory;
+    private ActivityViewBuilderFactoryInterface $activityViewBuilderFactory;
```

Comment:
This service has not been released yet, but it will be with `Sulu 2.3.1`. Until then, you either need to copy that [service](https://github.com/sulu/sulu/blob/2.3/src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Admin/View/ActivityViewBuilderFactory.php) into your project or require the `2.3@dev` version of sulu